app/Controllers/UpdateController.php

```
@@ -240,23 +242,119 @@ public function downloadBackup(Request $request, Response $response, mysqli $db)
+                'success' => true,
+                'message' => __('Ripristino completato'),
+                'safety_backup' => $result['safety_backup'],
+            ]);
+        }
+        return $this->jsonResponse($response, ['success' => false, 'error' => $result['error']], 500);
+    }
+
+    /**
+     * Persist the "include uploaded files in the pre-update backup" setting.
+     */
+    public function saveBackupSettings(Request $request, Response $response, mysqli $db): Response
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)